ReactiveCocoa/Swift/Action.swift

```diff
@@ -31,52 +31,52 @@ public final class Action<Input, Output, Error: ErrorType> {
 	public let errors: Signal<Error, NoError>
 
 	/// Whether the action is currently executing.
-	public var executing: AnyProperty<Bool> {
-		return AnyProperty(_executing)
+	public var isExecuting: AnyProperty<Bool> {
+		return AnyProperty(_isExecuting)
 	}
 
-	private let _executing: MutableProperty<Bool> = MutableProperty(false)
+	private let _isExecuting: MutableProperty<Bool> = MutableProperty(false)
 
 	/// Whether the action is currently enabled.
-	public var enabled: AnyProperty<Bool> {
-		return AnyProperty(_enabled)
+	public var isEnabled: AnyProperty<Bool> {
+		return AnyProperty(_isEnabled)
 	}
 
-	private let _enabled: MutableProperty<Bool> = MutableProperty(false)
+	private let _isEnabled: MutableProperty<Bool> = MutableProperty(false)
 
 	/// Whether the instantiator of this action wants it to be enabled.
-	private let userEnabled: AnyProperty<Bool>
+	private let isUserEnabled: AnyProperty<Bool>
 
 	/// This queue is used for read-modify-write operations on the `_executing`
 	/// property.
-	private let executingQueue = dispatch_queue_create("org.reactivecocoa.ReactiveCocoa.Action.executingQueue", DISPATCH_QUEUE_SERIAL)
+	private let executingQueue = DispatchQueue(label: "org.reactivecocoa.ReactiveCocoa.Action.executingQueue", attributes: DispatchQueueAttributes.serial)
 
 	/// Whether the action should be enabled for the given combination of user
 	/// enabledness and executing status.
-	private static func shouldBeEnabled(userEnabled userEnabled: Bool, executing: Bool) -> Bool {
+	private static func shouldBeEnabled(userEnabled: Bool, executing: Bool) -> Bool {
 		return userEnabled && !executing
 	}
 
 	/// Initializes an action that will be conditionally enabled, and create a
 	/// SignalProducer for each input.
-	public init<P: PropertyType where P.Value == Bool>(enabledIf: P, _ execute: Input -> SignalProducer<Output, Error>) {
+	public init<P: PropertyProtocol where P.Value == Bool>(enabling property: P, _ execute: (Input) -> SignalProducer<Output, Error>) {
```

Comment:
I think `enabling` is an incorrect translation here. This doesn't match the pattern from the Swift stdlib. I think `enabledIf` is a better name.